### PR TITLE
Rework build with esbuild

### PR DESCRIPTION
**Pull request by exdis**

### CODE REVIEW

File: src/views/editors-hint.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 /* eslint-env browser */
 
-import CodeMirror from '/gen/codemirror.js'; // FIXME: generated file to make it local
+import CodeMirror from 'codemirror'; // FIXME: generated file to make it local
```
Comment (lahmatiy) [inline src/views/editors-hint.js:8]: FIXME could be removed

File: src/views/editors.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import { createElement } from '../core/utils/dom.js';
 import { escapeHtml } from '../core/utils/html.js';
 import Emitter from '../core/emitter.js';
-import CodeMirror from '/gen/codemirror.js'; // FIXME: generated file to make it local
+import CodeMirror from 'codemirror'; // FIXME: generated file to make it local
```
Comment (lahmatiy) [inline src/views/editors.js:6]: FIXME could be removed

File: package.json [hunk 3]
```diff
@@ -47,8 +46,10 @@
         "babelify": "^10.0.0",
         "browserify": "^16.5.0",
         "cypress": "^5.3.0",
+        "esbuild": "^0.7.21",
         "eslint": "^6.8.0",
         "eslint-plugin-cypress": "^2.11.2",
+        "eswatch": "^0.6.0",
```
Comment (lahmatiy) [inline package.json:52]: How does it used?

Comment (exdis) [reply package.json]: That was experiment. Removed it.

**Pull request merged**